### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by tylerallan**
Practicing this.

### CODE REVIEW

**Comment by bwiernik** [comment]
🙌🏼

**Comment by tylerallan** [comment]
@diegodulanto @whitneykasenetz

File: hw01_exploring_data.html [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
...wUJEASeCQoRC%2F5TBwEjEv7K%2FsUFDwgLFQnlbm4TFRRWAS%2FTBhAAAAACAAAAAAT%2BBEwAHwA1AAABITIWHQEUBiMhIgYVERQWMyEyFh0BFAYjISImNRE0NgUBFhQHAQYmPQEjIiY9ATQ2OwE1NDYBXgGQFR0dFf4%2BKTs7KQHCFR0dFf5wpbm5AvEBRBAQ%2FrwQFvoVHR0V%2BhYETB0VZBUdOyn%2BDCk7HRVkFR25pQGQpbnp%2FuQOJg7%2B5A4KFZYdFcgVHZYVCgACAAAAAASwBLAAFQAxAAABITIWFREUBi8BAQYiLwEmNDcBJyY2ASMiBhURFBYzITI2PQE3ERQGIyEiJjURNDYzIQLuAZAVHRUObf7IDykPjQ8PAThtDgj%2B75wpOzspAfQpO8i7o%2F5wpbm5pQEsBLAdFf5wFQgObf7IDw%2BNDykPAThtDhX%2B1Dsp%2FgwpOzsplMj%2B1qW5uaUBkKW5AAADAA4ADgSiBKIADwAbACMAAAAyHgIUDgIiLgI0PgEEIg4BFB4BMj4BNCYEMhYUBiImNAHh7tmdXV2d2e7ZnV1dnQHD5sJxccLmwnFx%2FnugcnKgcgSiXZ3Z7tmdXV2d2e7ZnUdxwubCcXHC5sJzcqBycqAAAAMAAAAABEwEsAAVAB8AIwAAATMyFhURMzIWBwEGIicBJjY7ARE0NgEhMhYdASE1NDYFFTM1AcLIFR31FAoO%2FoEOJw3%2BhQ0JFfod%2FoUD6BUd%2B7QdA2dkBLAdFf6iFg%2F%2BVg8PAaoPFgFeFR38fB0V%2BvoVHWQyMgAAAAMAAAAABEwErAAVAB8AIwAACQEWBisBFRQGKwEiJj0BIyImNwE%2BAQEhMhYdASE1NDYFFTM1AkcBeg4KFfQiFsgUGPoUCw4Bfw4n%2FfkD6BUd%2B7QdA2dkBJ7%2BTQ8g%2BhQeHRX6IQ8BrxAC%2FH8dFfr6FR1kMjIAAwAAAAAETARLABQAHgAiAAAJATYyHwEWFAcBBiInASY0PwE2MhcDITIWHQEhNTQ2BRUzNQGMAXEHFQeLBwf98wcVB%2F7cBweLCBUH1APoFR37tB0DZ2QC0wFxBweLCBUH%2FfMICAEjCBQIiwcH%2FdIdFfr6FR1kMjIABAAAAAAETASbAAkAGQAjACcAABM3NjIfAQc
```
Comment (diegodulanto) [inline hw01_exploring_data.html]: Sounds good!

Comment (diegodulanto) [inline hw01_exploring_data.html]: Good work!

File: Hw_01/hw01_exploring_data.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    ## [325] "325" "326" "327" "328" "329" "330" "331" "332" "333" "334" "335" "336"
+    ## [337] "337" "338" "339" "340" "341" "342" "343" "344"
+
+``` r
+names(penguins)
+```
+
+    ## [1] "species"           "island"            "bill_length_mm"   
+    ## [4] "bill_depth_mm"     "flipper_length_mm" "body_mass_g"      
+    ## [7] "sex"               "year"
```
Comment (whitneykasenetz) [inline Hw_01/hw01_exploring_data.md:150]: Great job. It was a breeze to understand your work, and it looks like you followed all the instructions.

**Pull request merged**